R/zzz.R

```diff
@@ -35,6 +38,17 @@ set_transient_library <- function() {
     dir.create(transient_staging_path, FALSE, TRUE)
   }
   .libPaths(c(transient_staging_path, dir, .libPaths()))
+
+  # Finally, we initialize the transient library with some sane default symlinks.
+  with_real_packages(lockbox_library(), function(pkgpath) {
+    versions <- list.files(pkgpath)
```

Comment:
why not `versions <- as.package_version(list.files(pkgpath))` here? then you can get rid of the comment below